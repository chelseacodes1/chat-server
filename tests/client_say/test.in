SAY f1
SAY f1 hello
REGISTER user4 sunny4
LOGIN user4 sunny4
JOIN f1
SAY f1 3 more races to go
SAY f1 watching race 