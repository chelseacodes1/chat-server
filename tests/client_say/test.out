Error: Invalid use of SAY command
Error: user not logged in, cannot send message
RESULT REGISTER 1
RESULT LOGIN 1
RESULT JOIN f1 1
RECV user4 f1 3 more races to go
RECV user4 f1 watching race
